src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+input {
+  outline: none;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+
+  --primary-color: #e1e7ed;
```

Comment:
A common best practice is to define custom properties on the [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) pseudo-class, so that it can be applied globally across your HTML document:

```
:root {
  --primary-color: #e1e7ed;
}
```